anta/tests/cvx.py

```diff
@@ -87,3 +89,94 @@ def test(self) -> None:
         cluster_status = command_output["clusterStatus"]
         if (cluster_state := cluster_status.get("enabled")) != self.inputs.enabled:
             self.result.is_failure(f"Management CVX status is not valid: {cluster_state}")
+
+
+class VerifyCVXClusterStatus(AntaTest):
+    """Verifies the CVX Server Cluster status.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if all of the following conditions is met:
+        - CVX Enabled state is true   
+        - Cluster Mode is true
+        - Role is either Master or Standby
+        - peer_status matches defined state
+    * Failure: The test will fail if any of the success conditions is not met.
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.cvx:
+      - VerifyCVXClusterStatus:
+          enabled: true
```

Comment:
@sarunac do we really need the `enabled` input? it seems we are just checking if it i True (and this is one of your success conditions) so I feel like we should just remove this input.

Reply:
Ah I see your point. I will remove enabled and cluster_mode and update the examples and tests with appropriate docstring. Thank you!